### DISCUSSION TITLE
Publishing docs on readthedocs automatically using Github Actions

### DISCUSSION THREAD

**Discussion by josecm** [bao-project/bao-docs#11] [Ideas]
@danielRep Maybe we should start automating build the publishing of the documentation using Github Actions. What do you think?

**Comment by danielRep** [comment]
That seems reasonable. What should be the first steps? Do we already have a readthedocs account for the project?

**Reply by danielRep** [reply]
Some information:
https://docs.readthedocs.io/en/stable/tutorial/
https://www.sphinx-doc.org/en/master/tutorial/deploying.html#publishing-your-html-documentation

**Reply by josecm** [reply]
I'm not keen on using 3rd-party actions to build and publish the documentation. Can't we figure how to do it using the CLI and just run the commands in the usual CI container?

Yes we do have a readthedocs account setup for the project. I'll forward you the details through another channel.

**Reply by danielRep** [reply]
> I'm not keen on using 3rd-party actions to build and publish the documentation. Can't we figure how to do it using the CLI and just run the commands in the usual CI container?

Yeah, just need to familiarize with the readthedocs as soon as possible.
